.github/workflows/codeql.yml

```diff
@@ -47,12 +47,17 @@ jobs:
           config: |
             paths:
             - .github
+            sparse-checkout: |
+              .github
```

Comment:
Is this to scope to only the github folder? Do we not want any other validation? 

If we do want to scope it might be worth mentioning that here in a comment.